Jenkinsfile-publish-image

```diff
@@ -73,6 +74,7 @@ pipeline {
         stage('Build image') {
             steps {
                 script {
+                    sh "./run-cve-patcher.sh"
```

Comment:
What is the reason for having this as a separate file in the repo?

Reply:
We need this in 2 pipelines now, and we might need to include it in GitHub actions and other pipelines later, so it's just for convenience.

Reply:
Alright, that makes sense. I'll update the `core-dev` pipeline, as well.